clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/controllers/EcsClusterController.java

```diff
@@ -37,4 +39,10 @@ public EcsClusterController(EcsClusterProvider ecsClusterProvider) {
   public Collection<EcsCluster> getAllEcsClusters() {
     return ecsClusterProvider.getAllEcsClusters();
   }
+
+  @RequestMapping(value = {"/ecs/getEcsClusterDescriptions/{account}/{region}"})
```

Comment:
Since this is already define this as a GET operation, typically we don't add the operation type to the path. For instance, instead of adding getEcsClusterDescriptions to the path you might want to update it to be ecsClusterDescriptions like below:

Current: ("/ecs/getEcsClusterDescriptions/{account}/{region}")

to

Updated: ("/ecs/ecsClusterDescriptions/{account}/{region}")